src/components/About.js

```diff
@@ -1,8 +1,34 @@
-import React from "react";
+import React from 'react';
 
 const About = () => {
   return (
-    <h1> Hello This is About page</h1>
+    <main className="mx-5 my-3">
+      <h2 className="text-uppercase">
+        <strong>About CYF Quiz Project</strong>
+      </h2>
+      <p>Hello, and welcome to CYF quiz web app, so glad you're here.</p>
+      <p>
+        We are a group of graduates from CYF bootcamp, and decided to create
+        this application based on our experience as previous students to the
+        course, and as a graduates who are looking to give back to CYF
+        community. This app will help students to test their knowledge about
+        each given syllabuses, by taking each related quiz. With this app, we
+        aim to encourage our students to take new challenges, and build on the
+        confidence to try new thing every time. It is also a testament for us
+        the developers to present a valuable product that has been actively
+        benefiting the current students.
+      </p>
+      <h2 className="text-uppercase">
+        <strong>Meet the team</strong>
+      </h2>
+      <h6>The developers of this project are:</h6>
```

Comment:
This sentence doesn't seem like a heading to me, more like a paragraph. Let's change it to a paragraph

Reply:
I changed it.